tests/integration/test_remoted/test_socket_communication/data/test_cases/cases_syslog_message_parser.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Syslog_message_ipv6_test
```

Comment:
```suggestion
- name: syslog_message_ipv6
```

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3609/commits/b6c0aac937531f132aeb56fd1e5f4d49f589fd39)